cpp/roaring64map.hh

```diff
@@ -1712,6 +1708,9 @@ class Roaring64Map {
 
 /**
  * Used to go through the set bits. Not optimally fast, but convenient.
+ *
+ * strongly recommend not to implicitly construct this iterator:
+ * implicit construction may be prohibited in the future.
```

Comment:
I think it's possible that this will happen but not very likely, also the harm of implicit construction seems relatively small in the case of the iterator.